src/Entity/Transformer.php

```diff
@@ -177,11 +177,13 @@ public function loadBlock(array $content, PageInterface $page)
 
         $block->setPage($page);
         $block->setId($content['id']);
-        $block->setName($content['name']);
+        $block->setName($content['name'] ?? null);
         $block->setEnabled($content['enabled']);
-        $block->setPosition($content['position']);
+        if (isset($content['position'])) {
+            $block->setPosition($content['position']);
```

Comment:
I would do the same here. Use the ?? operator with null.

Reply:
i can do this for this branch because it is built against block bundle 3.x

but PageBundle 4.x will have problems against block bundle 4.x because:
https://github.com/sonata-project/SonataBlockBundle/blob/4.x/src/Model/BlockInterface.php#L46

see your MR for the problem: https://github.com/sonata-project/SonataPageBundle/pull/1501


so for this MR, i can fix it, and then let PageBundle 4.x be fixed later?